E2E/pageObjects/game.page.ts

```diff
@@ -16,7 +17,7 @@ const locator = {
     container: '$chess-pieces-container',
   },
   buttons: {
-    copyLink: '//button/p[contains(text(), "Copy Link")]',
+    copyLink: '//*[contains(text(), "Copy Link")]',
```

Comment:
There are two elements with "Copy link" text on page, doesn't this find both of them?